doc/how_to_guides/isaac_panda/launch/isaac_demo.launch.py

```diff
@@ -68,13 +68,29 @@ def generate_launch_description():
     )
 
     # Static TF
-    static_tf_node = Node(
+    world2robot_tf_node = Node(
         package="tf2_ros",
         executable="static_transform_publisher",
         name="static_transform_publisher",
         output="log",
         arguments=["0.0", "0.0", "0.0", "0.0", "0.0", "0.0", "world", "panda_link0"],
     )
+    hand2camera_tf_node = Node(
+        package="tf2_ros",
+        executable="static_transform_publisher",
+        name="static_transform_publisher",
+        output="log",
+        arguments=[
+            "0.04",
+            "0.0",
+            "0.04",
+            "0.0",
+            "0.0",
+            "0.0",
+            "panda_hand",
+            "sim_camera",
+        ],
+    )
```

Comment:
This static frame is not working because MoveIt is using the system clock and the images are using sim time. I'll see if I can work on a switch to properly enable all of the nodes launched in this file to `use_sim_time`